configs/opinicus_config

```diff
@@ -5039,14 +5039,14 @@ CONFIG_MISC_FILESYSTEMS=y
 # CONFIG_JFFS2_FS is not set
 # CONFIG_UBIFS_FS is not set
 # CONFIG_CRAMFS is not set
-CONFIG_SQUASHFS=m
+CONFIG_SQUASHFS=y
 CONFIG_SQUASHFS_FILE_CACHE=y
 # CONFIG_SQUASHFS_FILE_DIRECT is not set
 CONFIG_SQUASHFS_DECOMP_SINGLE=y
 # CONFIG_SQUASHFS_DECOMP_MULTI is not set
 # CONFIG_SQUASHFS_DECOMP_MULTI_PERCPU is not set
 # CONFIG_SQUASHFS_XATTR is not set
-CONFIG_SQUASHFS_ZLIB=y
+# CONFIG_SQUASHFS_ZLIB is not set
```

Comment:
did you purposly disable zlib? i'm fine with it disabled as it potentially saves some space I suppose, but it does not match your commit message.

Reply:
yess we dont need id